.github/workflows/tests-schedule.yml

```diff
@@ -1,14 +1,41 @@
-name: Daily scheduled extended E2E tests
+name: Daily scheduled Tests
 
 on:
   push:
     branches: [force_test, release-*]
   schedule:
     - cron: "6 0 * * *"
 
+defaults:
+  run:
+    working-directory: ./cli
+
 jobs:
-  build:
-    name: Build
+  golang:
```

Comment:
are we sure we don't want to keep the build, lint,format,vet, etc in the scheduled tests as well? I know they are tested in the branches but I always think of the scheduled tests as a superset of the tests.

Reply:
IMHO Scheduled tests should test parts of code that can change without changing things on `main/master` branch. Mostly this is related to external dependencies. For lint, vet, format, etc. code can only be changed with PR process and as such testing it during pre-merge and post-merge is sufficient. Adding those steps in a scheduled job when there is no possibility of code change only increases perceived test flakiness and increases fatigue, as such I would like to limit periodic tests to bare minimum.

Reply:
Makes sense